docs/features/stem_export.md

```diff
@@ -88,21 +89,23 @@ This menu allows you to start a stem export and configure various settings relat
 
 ### I have a track that won't export
 
-#### Scenario #1: Special characters in the track name 
+#### Scenario: Special characters in the track name 
 
 One user reported that they were unable to export a track even though the stem export indicated that the track had been exported.
 
 Solution: 
 
 Check that the track name doesn't have any special characters. In this case, the user had a track called << Organ >>. Removing the characters "<< >>" and just naming the track "Organ" allowed the track to be exported.
 
-#### Scenario #2: The track has an exceptionally long tail that doesn't not drop to silence
+### I have a track that takes much longer to export than others
 
-If you are using the `EXPORT TO SILENCE` feature, it may not automatically export your track because the track never becomes sufficiently silent. Thus, you will see that playback was turned off but the recording button continues to blink rapidly because it is still waiting for silence before stopping the recording.
+#### Scenario: The track has an exceptionally long tail that doesn't not drop to silence
 
-In this case, you will need to press `BACK` to cancel the stem export. If you press `BACK` after the track's loop length is finished, it will save the stem, however the stem's length may not match the loop length.
+If you are using the `EXPORT TO SILENCE` feature, it may take longer for your track to export because the recorder is waiting for your track to become silent. Thus, you will see that playback has turned off but the record button continues to blink rapidly because it is still waiting for silence before stopping the recording.
 
-Possible solution:
+As a safety measure, if your track does not become silent within 60 seconds of playback stopping, then the recording will stop automatically.
+
+If you do not want to wait potentially 60 seconds, you will need to press `BACK` to cancel the stem export. If you press `BACK` after the track's loop length is finished, it will save the stem, however the stem's length may not match the loop length.
```

Comment:
why is this? Intuitively it feels like we should be able to set the correct loop length anyway and only the ending would be shortened